dowhy/causal_model.py

```diff
@@ -318,15 +318,15 @@ def refute_estimate(self, estimand, estimate, method_name=None, **kwargs):
         res = refuter.refute_estimate()
         return res
 
-    def view_model(self, layout="dot"):
+    def view_model(self, layout="dot", size=(8, 6), file_name="causal_model"):
         """View the causal DAG.
 
         :param layout: string specifying the layout of the graph.
```

Comment:
Can you add the two new parameters to the docstring here, in the same `:param:` format? since this is a user-facing function, it will automatically show up in the docs then.

Reply:
Sure, I've done that now.